using-kubecost/navigating-the-kubecost-ui/teams.md

```diff
@@ -23,79 +22,254 @@ persistentVolume:
+5. Define which pages can be viewed by users with the role. If any `Monitoring` pages are shown, you may additionally define a filter that will apply to the contents of this page.
 
-User roles can always be optionally configured using the UI or by adjusting the Helm flags in your *values.yaml*. There is no reequirement to choose one or the other.
+### Adding a team through the Kubecost UI
 
-Kubecost RBAC follows the principle of most privilege. If a user is added to multiple teams with different roles, they will assume the role with the highest authority unless Allocation Filters are used to limit role functions.
+![Adding a Team](/images/rbac-teams-team-creation-example.png)
 
-Authentication is handled by the cost-model and Aggregator pods.
+To add a team:
 
-## Debugging
+1. Click the `New Team` button on the teams page.
+2. Add a team name. Note that this must be unique, and multiple teams cannot share the same name.
+3. Define user claims. For more information on what a claim is, see the [Claims](teams.md#claims) section above. __Note that any overlap between a user's claim data from the IdP and a team's claims will assign the user to the team, even if there are other claims specified in the Team.__
+4. Add one or more roles.
 
-For help with troubleshooting, follow this guide for diagnostic assistance:
+On login, the user with the associated claims will be assigned the permissions of their team(s) role(s).
+
+Note that, depending on the claims set, a user may have multiple teams. Also note that a team can have multiple roles. For more info, see the [Multiple teams](teams.md#multiple-teams) section below.
+
+## Multiple teams
+
+If a user has multiple roles (either through having multiple teams, a team with multiple roles, or both), they will be assigned an __additive__ union of the permissions defined by each role.
+
+### Filters
+
+Asset, Allocation, and Cloud Cost filters are ANDed together within a role, and then ORed together within both a set of roles in a team and a set of teams for a user.
+
+For example, if a user is assigned both roles `Role 1` and `Role 2` with the following Allocation filters:
+
+```
+Role 1:
+cluster IS cluster-1, cluster-2
+namespace IS kubecost
+
+Role 2:
+cluster IS cluster-3
+namespace IS dev
+```
+
+The user will gain a combined filter of `((cluster = cluster-1 OR cluster = cluster-2) AND namespace = kubecost) OR (cluster = cluster-3 AND namespace = dev)`.
+
+### Permissions level
+
+Permissions level will respect only the highest level out of all roles for all teams assigned to a user. The order of precedence is `Admin` > `Editor` > `Read Only`.
+
+### Pages
+
+Disabled/enabled pages will enable any page marked as such in any one of a user's roles. Otherwise, the page is designated as disabled. Essentially, if any of the user's roles has a page enabled, that page will be enabled for the user. Only if none of the user's roles have a page enabled is it disabled.

```

Comment:
This is slightly confusing to read to me and I'm having trouble parsing it. Essentially you're saying that a page is considered disabled if it remains unchecked. If a user has multiple roles then all of those roles must have the same page unchecked otherwise, because permissions are additive, any role which has the page enabled will allow the user access. Is that right?

Reply:
Yes. I reworded this slightly to make it (hopefully) a little less confusing.